.github/docs/HostedWidgetVSCustomBuilderSolution.md

```diff
@@ -0,0 +1,100 @@
+# Hosted Widget vs Custom Builder Solution Documentation
+
+## Overview
+
+[Amazon Connect](https://aws.amazon.com/connect) is an omni-channel contact center solution which includes support for chat (real time messaging between business and end-customer). An out-of-box fully managed/hosted communications widget is provided for the customer chat UI. This enables customer to copy/paste a simple code snippet to set up in minutes.
+
+Alternatively, AWS Customer can also build a custom customer-facing chat UI with ChatJS (an open-source JavaScript library published to npm). This is a wrapper around the AWS Javascript SDK, which calls the [Amazon Connect Participant Service Public APIs](https://docs.aws.amazon.com/connect-participant/latest/APIReference/Welcome.html). The hosted widget chat interface code has also been open sourced, and is available for customer to fork to avoid re-writing boilerplate code. 
+
+AWS Customers have multiple options for integrating chat into their contact center depending on use-case and size of the business.
+
+## Hosted Widget
+
+Simple copy/paste snippet solution to render customer chat interface on your website, fully hosted and managed by Amazon Connect. Always receive the latest features without redeploying.
+
+### Prerequisites
+- An Amazon Connect Instance ([guide](https://docs.aws.amazon.com/connect/latest/adminguide/amazon-connect-instances.html))
+
+### Usage:
+
+```html
```

Comment:
nit: do we need to make this copyable? Some people may try to copy this snippet to run on their website, and they may face CSP issues or other errors. 
If we want to provide a live example I think we need to have some troubleshootings otherwise we can just mention this is an example of snippet which will not work without certain setup

Reply:
+1

Reply:
Yeah this is not intended to be copyable could catch. I can add comment to the snippet in short follow-up.

I don't think we should provide working example, since they cannot access CCP anyways.